assets/themes/restart/css/style.css

```diff
@@ -9162,4 +9162,22 @@ blockquote p {
 .div-report-btn button {
   color: #428bca;
   border: none;
-}
\ No newline at end of file
+}
+.tell-more {
```

Comment:
Rename this class to `page-jobad-modal-tell-more`

Reply:
ok